helm_deploy/hmpps-interventions-ui/templates/_envs.tpl

```diff
@@ -10,26 +10,20 @@ env:
   - name: CLIENT_ID
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: CLIENT_ID
+        name: hmpps-auth
+        key: client-id
 
   - name: CLIENT_SECRET
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: CLIENT_SECRET
-
-  - name: APPINSIGHTS_INSTRUMENTATIONKEY
-    valueFrom:
-      secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: APPINSIGHTS_INSTRUMENTATIONKEY
+        name: hmpps-auth
+        key: client-secret
 
   - name: SESSION_SECRET
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: SESSION_SECRET
+        name: session
```

Comment:
_not a blocker_

Should we scope these names somehow so it's obvious they belong to the UI? The kubernetes namespace we use is shared between the UI, and the service deployments, so it may get confusing after a while.

Reply:
oh damn - i thought we had seperate namespaces. i need to fix up some of the secret names in that case.

and yes to scoping these types of var.

Reply:
To be honest, I made an (invisible) assumption there that we don't need separate ones -- sorry about that. If it makes more sense we can definitely have everything on its own?

Reply:
i'm fine with it as it is - especially since k8s secrets are really just a stop-gap(?)

Reply:
I think, ultimately, the deployment _must_ populate Kubernetes secrets. I think we'll change the mechanism of _how_ they get there (currently: manual, later: CI pulls them down from AWS then pushes it through Helm) so names will still be a concern. However, this isn't a blocker for this PR 👍